schema/partner.js

```diff
@@ -9,69 +9,76 @@ import {
   GraphQLNonNull,
   GraphQLInt,
   GraphQLList,
-  GraphQLBoolean
+  GraphQLBoolean,
 } from 'graphql';
 
-let PartnerType = new GraphQLObjectType({
+const PartnerType = new GraphQLObjectType({
   name: 'Partner',
   fields: () => ({
-    cached: cached,
+    cached,
     _id: {
-      type: GraphQLString
+      type: GraphQLString,
     },
     id: {
-      type: GraphQLString
+      type: GraphQLString,
     },
     name: {
-      type: GraphQLString
+      type: GraphQLString,
     },
     type: {
-      type: GraphQLString
+      type: GraphQLString,
     },
     href: {
       type: GraphQLString,
-      resolve: ({ default_profile_id }) => `/${default_profile_id}`
+      resolve: ({ default_profile_id }) => `/${default_profile_id}`,
     },
     is_linkable: {
       type: GraphQLBoolean,
-      resolve: ({ default_profile_id, default_profile_public}) => default_profile_id && default_profile_public
+      resolve: ({ default_profile_id, default_profile_public }) => {
+        return default_profile_id && default_profile_public;
+      },
     },
     initials: {
       type: GraphQLString,
-      resolve: ({ name }) => _.take(name.replace(/[^A-Z]/g, ''), 3).join('')
+      resolve: ({ name }) => _.take(name.replace(/[^A-Z]/g, ''), 3).join(''),
     },
     default_profile_id: {
-      type: GraphQLString
+      type: GraphQLString,
     },
     profile: {
       type: Profile.type,
-      resolve: ({ default_profile_id }) => gravity(`profile/${default_profile_id}`)
+      resolve: ({ default_profile_id }) => gravity(`profile/${default_profile_id}`),
     },
     type: {
-      type: GraphQLString
+      type: GraphQLString,
+      deprecationReason: 'Avoid naming fields `type`',
```

Comment:
This results in a parse error currently:

![](http://static.damonzucconi.com/_capture/MdZw1hL1sM.png)

I'm just deprecating it and then I'll change the caller in the show2 routes. Then we can remove it and run eslint as a buildstep